Module14/07_years/main.py

```diff
@@ -0,0 +1,24 @@
+def print_years_with_3_same_digits(low_year, up_year):
+    for year in range(low_year, up_year + 1):
```

Comment:
What would be if the first parameter is greater or equal to the second?

Reply:
Error from for loop